runtime-desktop/fontconfig/autobuild/overrides/etc/fonts/conf.d/70-no-bitmaps.conf

```diff
@@ -1 +1 @@
-../conf.avail/70-no-bitmaps.conf
\ No newline at end of file
+/usr/share/fontconfig/conf.avail/70-no-bitmaps.conf
```

Comment:
This should be `../usr/share/...`.

Reply:
if it's `../usr/share/..`, it will becomes `/etc/fonts/usr/share` when it is accessed.

Reply:
Not how symlinks work. Instead of using overrides, do this in `beyond`.

```
ln -sv ../usr/share/fontconfig/conf.avail/70-no-bitmaps.conf /etc/fonts/conf.d/70-no-bitmaps.conf
```

Reply:
use upstream's relative path script patch, resolved